p2panda-rs/src/wasm/operation.rs

```diff
@@ -70,16 +70,16 @@ impl OperationFields {
                 Ok(())
             }
             "relation" => {
-                let relation: Relation = jserr!(deserialize_from_js(value), "Invalid object");
+                let relation: OperationValueRelation =
```

Comment:
Shouldn't we add cases for `pinned_relation` and `pinned_relation_list` here?

Reply:
Its quite nice: they get automatically deserialized :star_struck: 

If a `relation` is an array its automatically a `PinnedRelation`, otherwise `Relation`. If `relation_list` is an array its automatically a `RelationList`, if its an array of arrays, its a `PinnedRelationList`. Serde magic!

Reply:
This is why `OperationValueRelationList` and `OperationValueRelation` is so involved actually. To have these "union" types around all the variations, but still be strict in the types (you cant be strict with enums).

Reply:
Ahh it's the _other_ relation here. Yes, I think it's time for me to call it a day haha.